quattor/functions/strings.pan

```diff
@@ -0,0 +1,29 @@
+declaration template quattor/functions/strings;
+
+include { 'pan/functions' };
+
+#version comparison function (should not be here)
+function comparever = {
```

Comment:
Yes of course I can! That was exactly what I was looking for before deciding to write one of my own. I've tested it and I can reuse the pkg_compare_version function as it is for the moment (and, if later needed, I'll modify it).  I think we can just cancel this pull request and I'll modify the templates in the the pull request #154 so that they use the pkg_compare_version function